spec/requests/withdraw_spec.rb

```diff
@@ -1,57 +1,81 @@
 # frozen_string_literal: true
 
 RSpec.describe "Withdraw" do
-  let(:edition) { create(:edition, :published, :political, :past_government) }
+  let(:managing_editor) { create(:user, managing_editor: true) }
+  let(:published_edition) { create(:edition, :published) }
+  let(:history_mode_edition) { create(:edition, :published, :political, :past_government) }
 
   describe "POST /documents/:document/withdraw" do
+    it "withdraws the edition" do
+      stub_publishing_api_unpublish(published_edition.content_id, body: {})
+      login_as(managing_editor)
+
+      post withdraw_path(published_edition.document), params: { public_explanation: "Just cos" }
+      follow_redirect!
+
+      withdrawal = published_edition.reload.status.details
+      expect(response.body).to include(
+        I18n.t!("documents.show.withdrawn.title",
+                document_type: published_edition.document_type.label.downcase,
+                withdrawn_date: withdrawal.created_at.strftime("%-d %B %Y")),
+      )
+    end
+
     context "when the edition is in history mode" do
       it "lets users holding manage_live_history_mode permisssion withdraw the edition" do
```

Comment:
Not your code, but noticed a typo here (`permisssion`). Happens in a few places in this file, could you do a mass find-and-replace in a separate commit please?

EDIT: looks like it also applies in `unwithdraw_spec.rb` - maybe do the find-and-replace on the whole repo!

Reply:
I got too eager with the s's 🤦‍♂